SofascoreAcademy/SofascoreAcademy/Controllers/EventsViewController.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  EventsViewController.swift
+//  SofascoreAcademy
+//
+//  Created by Matija Pavicic on 21.03.2024..
+//
+
+import Foundation
+import SnapKit
+import SofaAcademic
+import UIKit
+
+class EventsViewController: UIViewController {
+    
+    var tournaments = TournamentHeaderModel.sampleData
+    var events = EventModel.sampleData
+    
+    let tableView: UITableView = .init()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        view.backgroundColor = .white
+        
+        view.addSubview(tableView)
+        
+        tableView.dataSource = self
+        tableView.delegate = self
+        tableView.register(EventTableCell.self, forCellReuseIdentifier: "EventTableCell")
+        tableView.register(TournamentTableHeader.self, forHeaderFooterViewReuseIdentifier: "TournamentTableHeader")
+        
+        tableView.separatorStyle = .none
+        
+        tableView.snp.makeConstraints {
+            $0.top.equalTo(view.safeAreaLayoutGuide)
+            $0.leading.trailing.bottom.equalToSuperview()
+        }
+        tableView.reloadData()
+    }
+}
+
+extension EventsViewController: UITableViewDataSource, UITableViewDelegate {
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return tournaments.count
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        switch section {
+        case 0:
+            events[tournaments[section].countryName]?.count ?? 0
+        case 1:
+            events[tournaments[section].countryName]?.count ?? 0
+        case 2:
+            events[tournaments[section].countryName]?.count ?? 0
+        case 3:
+            events[tournaments[section].countryName]?.count ?? 0
+        case 4:
+            events[tournaments[section].countryName]?.count ?? 0
+        default:
+            0
+        }
+    }
```

Comment:
```suggestion
        events[tournaments[section].countryName]?.count ?? 0
```

Jel nije ovo dovoljno?